app/poros/movie.rb

```diff
@@ -0,0 +1,21 @@
+class Movie
+  attr_reader :id,
+              :title,
+              :overview,
+              :runtime,
+              :vote_count,
+              :vote_average,
+              :genres,
+              :poster_path
+
+  def initialize(data)
+    @id = data[:id]
+    @title = data[:title]
+    @summary = data[:overview]
+    @runtime = data[:runtime]
```

Comment:
Might need to change what we are initializing bc we are not currently getting access to all these things in our api call.